src/lime/ui/Gamepad.hx

```diff
@@ -43,19 +52,35 @@ class Gamepad
 	}
 
 	/**
-		@param	lowFrequencyRumble	The intensity of the low frequency (left)
-		rumble motor, from 0 to 1.
-		@param	highFrequencyRumble	The intensity of the high frequency (right)
+		@param	lowFrequencyRumble	The intensity of the low frequency (strong)
 		rumble motor, from 0 to 1.
+		@param	highFrequencyRumble	The intensity of the high frequency (weak)
+		rumble motor, from 0 to 1. Will be ignored in situations where only one
+		motor is available.
 		@param	duration	The duration of the rumble effect, in milliseconds.
 	**/
 	public inline function rumble(lowFrequencyRumble:Float, highFrequencyRumble:Float, duration:Int):Void
 	{
 		#if (lime_cffi && !macro)
 		NativeCFFI.lime_gamepad_rumble(this.id, lowFrequencyRumble, highFrequencyRumble, duration);
 		#elseif (js && html5)
-		// TODO: HTML5 Rumble
-		return;
+		var actuator:Dynamic = (untyped __jsGamepad.vibrationActuator) ? untyped __jsGamepad.vibrationActuator
```

Comment:
should this use [js.Syntax.code](https://haxe.org/manual/target-syntax.html), instead?

Reply:
That's recommended, yes. Though since it isn't available in Haxe 3, we'd need to use `#if haxe4 js.Syntax.code #else untyped __js__ #end (...)`.

I think the main reason I went ahead with this version is [the `Joystick` class does the same thing](https://github.com/openfl/lime/blob/6d437283afa9c8d40c6bcce9f06c107a1bf2d9fa/src/lime/ui/Joystick.hx#L61). If the shortcut was going to cause problems, those problems were already going to happen.

But I do support migrating both at some point, either now or when we eventually drop Haxe 3 support.